core/conventional-commits/lib/recommend-version.js

```diff
@@ -35,8 +49,15 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath, tagPrefix }) {
         // we still need to bump _something_ because lerna saw a change here
         const releaseType = data.releaseType || "patch";
 
-        log.verbose(type, "increment %s by %s", pkg.version, releaseType);
-        resolve(semver.inc(pkg.version, releaseType));
+        if (prereleaseId) {
```

Comment:
`prereleaseId` both activates prerelease recommendations and provides the prerelease ID to be used.